cypress/fixtures/minimal_metadata_edit.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
With these new fixtures we now store identical test schemas in four different places:

```
server/scripts/example_schemas
lib/python/tests/resources
__tests__/example_models/minimal_model
cypress/fixtures
```

We should use existing sources, instead of duplicating them yet again.

Reply:
The test schemas do have slight differences, but I'm wondering if we can fetch the content of the test schemas in the cypress test and edit them in there, therefore only having one overall?

Either way it's common practice to keep any test material inside the fixtures directory, so not sure how comfortable I am moving it. We could make a directory for any shared example schemas for tests (python and cypress - we'll be removing __tests__ soon). If we want to use them all in the same place I'd say we should make a new directory at the route level

Reply:
I've removed them from the python test folder (they now use python fixtures directory).
The __tests__ directory will be deleted once Cypress tests are finished. That just leaves them in two places, the /server/scripts dir, and /cypress/fixtures. I've ammended the tests to only use one metadata file though and edit the metadata for any test specific changes.